src/main/webapp/app/pages/genomicPage/GenomicPage.tsx

```diff
@@ -52,13 +55,32 @@ export default class HgvsgPage extends React.Component<HgvsgPageProps> {
   constructor(props: any) {
     super(props);
 
+    if (props.routing.location.pathname.startsWith(PAGE_ROUTE.HGVSG)) {
```

Comment:
Should we can make the checks case insensitive, otherwise `/HGVSG/:query` would not be annotated.

The other place where I see this: https://www.oncokb.org/YEAR-END-SUMMARY. The tab is not selected.

Reply:
Good point, but I think all of our pages URL are case sensitive. I can fix the `startsWith` usage, we could create a separate issue to discuss whether making URL case insensitive makes sense.